src/SldStyleParser.ts

```diff
@@ -340,6 +340,16 @@ export class SldStyleParser implements StyleParser {
           break;
       }
     });
+    
+    fillParams.forEach((param: any) => {
+      switch (param.$.name) {
+        case 'fill-opacity':
+          markSymbolizer.fillOpacity = parseFloat(param._);
+          break;
+        default:
+         break;
+      }
+    });
```

Comment:
As your just searching for one specific parameter `forEach` and `switch` seem to be overdose.
Maybe you find a way to use `_get` from lodash like we did above or [`find`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/find) on the array.

Also the property on the symbolizer is wrong. It is `opacity` not `fillOpacity`. Compare [BaseSymbolizer](https://github.com/terrestris/geostyler-style/blob/master/index.d.ts#L94).